util/progress/progressui/display.go

```diff
@@ -164,7 +269,6 @@ type vertex struct {
 	logsOffset    int
 	logsBuffer    *ring.Ring // stores last logs to print them on error
 	prev          *client.Vertex
-	events        []string
```

Comment:
This appeared to be dead code. While this variable was used, it never appeared to be set except in this package where it was cleared after it was read from.